cdap-common/src/main/java/io/cdap/cdap/common/http/AuthenticationChannelHandler.java

```diff
@@ -56,6 +56,8 @@ public class AuthenticationChannelHandler extends ChannelDuplexHandler {
   private static final String EMPTY_USER_IP = "CDAP-empty-user-ip";
   static final String AUDIT_LOG_REQ_BUILDER_ATTR = "AUDIT_LOG_REQ_BUILDER";
   static final String AUDIT_LOG_USER_IP_ATTR = "AUDIT_LOG_USER_IP";
+  static final String USER_ID_ATTR = "USER_ID";
+  static final String USER_CREDENTIAL_ATTR = "USER_CREDENTIAL";
```

Comment:
Add CDAP prefix to the names. That's a netty channel-wide attribute store, it's better to use prefix for not to have any collisions.

Reply:
done, thanks!